service-templates/apigw-lambda-svc/v1/instance_infrastructure/README.md

```diff
@@ -0,0 +1,78 @@
+# Instance
+
+## Local Development & Testing
+
+Copy the provided `example.terraform.tfvars` locally and rename to `terraform.tfvars`.
+
+Add, update, modify any variables required for testing within the `terraform.tfvars` file.
+
+Provision the resources using:
+
+```bash
+$ terraform init
+$ terraform plan
+$ terraform apply
+```
+Note that this example may create resources which cost money. Run `terraform destroy` when you no longer need these resources.
+
+```
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
These lines are tied to the `terraform_docs` pre-commit hook. Anything between the opening and closing is automatically updated when you run `pre-commit run -a`